### ISSUE TITLE
Centralized logging

### ISSUE THREAD

**Issue by thekeenant** [Avicus/Compendium#2]
Compendium should store loggers and have a command to change the logging level of each?

**Comment by AustinLMayes** [comment]
OR Just log to a service.

There are many open-source ones that we can host. I personally think we need a more organized way of catching errors. Could also implement into web to catch web errors.

General methods could be in Compendium though.

**Comment by thekeenant** [comment]
True, that too, since those services listen to logging systems like java.util.logging and Log4j, it's necessary to first setup logging properly in the plugins.
